src/main/java/com/suse/salt/netapi/parser/ResultSSHResultTypeAdapterFactory.java

```diff
@@ -85,7 +85,7 @@ public static boolean isResultSSHResult(Type type) {
                                         .flatMap(SaltErrorUtils::deriveError)
                                         .orElse(
                                                 new SaltSSHError(result.getRetcode(),
-                                                        result.getStderr().orElse(""))
```

Comment:
Is it possible that it return emptry string in case if "stderr" is present in the response, but blank?

Reply:
Agree and looking to the salt code most probably only one of them will contain something in case of error, and I like the idea to combine both to show :+1: